--- a/hologram.asl
+++ b/hologram.asl
@@ -2740,7 +2740,7 @@
 }
 
 >>FUNCmodPatch () { //keep here as patching only works on original/vanilla files
-	Set £TestModPatch "patched3b" //change to "patched" at the diff's patch file !
+	Set £TestModPatch "patched2b" //change to "patched" at the diff's patch file !
 	++ §testsPerformed
 	Set £TestsCompleted "~£TestsCompleted~, ~^debugcalledfrom_0~"
 	RETURN
